Base/usr/share/man/man1/chown.md

```diff
@@ -0,0 +1,31 @@
+## Name
+
+chown - Change file owner / group
+
+## Synopsis
+
+```**sh
+$ chown [user][:group] files...
+```
+
+## Description
+
+`chown` changes owner of specified files to `user`, and owning group to `group`. If `group` is not specified, it is leaved unchanged.
+
+**NOTE**: The caller must be a superuser to use this command.
```

Comment:
That's not necessarily true. A non-root user can use `chown` to change the group to one of their other groups.